grakn-core/src/main/java/ai/grakn/concept/Rule.java

```diff
@@ -21,33 +21,21 @@
 import ai.grakn.graql.Pattern;
 
 import javax.annotation.CheckReturnValue;
+import javax.annotation.Nonnull;
 import java.util.stream.Stream;
 
 /**
  * <p>
- *     A rule which defines how implicit knowledge can extracted.
+ *     An ontological element used to model and categorise different types of {@link Rule}.
```

Comment:
This description is wrong. Now these are actually rules themselves. Probably better to take the old description of `Rule` (and update it not to say LHS etc. 😛)

Reply:
🍌